lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -28,6 +28,8 @@ import { AuthenticationResult } from "../response/AuthenticationResult";
 import { EventCallbackFunction } from "../event/EventMessage";
 import { ClearCacheRequest } from "../request/ClearCacheRequest";
 import { EndSessionPopupRequest } from "../request/EndSessionPopupRequest";
+import { NestedAppAuthController } from "../controllers/NestedAppAuthController";
```

Comment:
just confirming -- because these imports are in the main PCA.ts file, they _may_ increase bundle size for folks that don't have tree shaking configured correctly. Since msal-browser specifies sideEffects=false, most cases should be smart enough to shake this out of the bundle if they don't call  `createNestablePublicClientApplication` directly, but just fyi

Reply:
This is acceptable IMO. If you haven't configured treeshaking you likely don't care about size anyway.

Reply:
+1 and we will try doing whatever size optimization we can eventually but the focus is to make sure at least tree shaking works as intended and will be improved.